site/app/config/services.yml

```diff
@@ -94,3 +97,6 @@ services:
     App\RepoCrawler\GitRepoCrawler:
         tags:
             - 'app.repo_crawler'
+
+    # Register Mgilet NotificationManager as a service to be used in WebNotificationConsumer
+    Mgilet\NotificationBundle\Manager\NotificationManager: '@mgilet.notification'
```

Comment:
We discussed this before: This commit is titled " setup Rabbit MQ architecture with respective consumers ". the mgilet bundle is not needed here, and should not appear anywhere in this commit.

Reply:
This is a mistake I overlooked accidentally